envfile/envfile.go

```diff
@@ -0,0 +1,107 @@
+package envfile
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/bitrise-io/bitrise/v2/log"
+	"github.com/bitrise-io/colorstring"
+	envmanModels "github.com/bitrise-io/envman/v2/models"
+	"gopkg.in/yaml.v3"
+)
+
+// DefaultEnvfilePathEnv is the env var that points to the (platform-dependent) envfile location.
+const DefaultEnvfilePathEnv = "BITRISEIO_ENVFILE_PATH"
+
+type EnvFile struct {
+	Envs       map[string]string `yaml:"envs"`
+	ErasedEnvs []string          `yaml:"erased_envs"`
+}
+
+// GetEnv returns the true value of an env var, even if it was not exposed to the CLI process because of its size.
```

Comment:
Based on the PR description this:
`even if it was not exposed to the CLI process because of its size.`
should be somthing  like this:
`even if it was erased because of its size.`